src/modules/ethereum/types/TransactionResponsePlain.ts

```diff
@@ -0,0 +1,106 @@
+import type { HexString } from 'ethers/lib.esm/utils/data';
+
+/** Serializable subset of public props of {ethers.TransactionResponse} */
+export interface TransactionResponsePlain {
```

Comment:
rename to `SerializableTransactionResponse`?

Reply:
Yes, please :)